17_kata.py

```diff
@@ -0,0 +1,15 @@
+# https://www.codewars.com/kata/54bb6f887e5a80180900046b/train/python
+
+def longest_palindrome(s):
+    if len(s) > 1:
+        empty_list = []
```

Comment:
Here I would also suggest to rename this variable. The list is obviously empty, but only initially. Later you store things in it, and from the get-go, it's difficult to say what is stored in this list.